lib/plugins/package/lib/packageService.js

```diff
@@ -52,9 +52,12 @@ module.exports = {
   packageAll() {
     const exclude = this.getExcludes();
     const include = this.getIncludes();
-    const zipFileName = this.provider.naming.getServiceArtifactName();
+    const zipFileName = `${this.serverless.service.service}.zip`;
```

Comment:
Maybe overkill but should we move this into a `provider.naming` function as well?